crates/aptos-genesis/src/config.rs

```diff
@@ -32,15 +31,19 @@ pub struct Layout {
     pub users: Vec<String>,
     /// ChainId for the target network
     pub chain_id: ChainId,
-    /// Whether to allow validators to join post genesis
+    /// Whether to allow new validators to join the set after genesis
     #[serde(default)]
     pub allow_new_validators: bool,
-    /// Initial lockup period for genesis validators
-    #[serde(default)]
-    pub initial_lockup_period_duration_secs: u64,
-    /// Initial balances for the target network
-    #[serde(default)]
-    pub initial_balances: HashMap<AccountAddress, u64>,
+    /// Minimum stake to be in the validator set
+    pub min_stake: u64,
+    /// Minimum number of seconds to lockup staked coins
+    pub min_lockup_duration_secs: u64,
```

Comment:
Can you add a max_lockup_duration_secs as well?

Reply:
Okay, I had them but it wasn't in the reqs, so I'll fix it.